Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.centos.org/sclo/nodejs-4-centos7:latest
```

Comment:
While testing the image on minishift I have noticed that the application does not start because it is using an old node.js version can you move to `nodejs-6-centos7:latest` which is node.js 6 based?